src/main/java/com/github/dockerjava/api/model/Container.java

```diff
@@ -75,6 +84,12 @@
     private ContainerHostConfig hostConfig;
 
     /**
+     * @since ~{@link RemoteApiVersion#VERSION_1_23}
+     */
+    @JsonProperty("Mounts")
+    private List<InspectContainerResponse.Mount> mounts;
```

Comment:
Well I can't tell what the actual response is, in `List` https://docs.docker.com/engine/api/v1.29/#operation/ContainerList
The sample response is:
```
"Mounts": [
{
"Name": "fac362...80535",
"Source": "/data",
"Destination": "/data",
"Driver": "local",
"Mode": "ro,Z",
"RW": false,
"Propagation": ""
}
]
```
but the `RESPONSE SCHEMA` says:
```
Target | string Container path.
Source | < anything > Mount source (e.g. a volume name, a host path).
Type | string "bind" "volume" "tmpfs"
ReadOnly | boolean Whether the mount should be read-only.
Consistency | string The consistency requirement for the mount: default, consistent, cached, or delegated.
BindOptions | BindOptions Optional configuration for the bind type.
VolumeOptions | VolumeOptions Optional configuration for the volume type.
TmpfsOptions | TmpfsOptions Optional configuration for the tmpfs type.
```
These don't even match? For what it's worth, I get response similar to sample when running locally.
While for `Inspect` response https://docs.docker.com/engine/api/v1.29/#operation/ContainerInspect they match, the sample:
```
"Mounts": [
{
"Name": "fac362...80535",
"Source": "/data",
"Destination": "/data",
"Driver": "local",
"Mode": "ro,Z",
"RW": false,
"Propagation": ""
}
]
```
 `RESPONSE SCHEMA`:
```
Type | string
Name | string
Source | string
Destination | string
Driver | string
Mode | string
RW | boolean
Propagation | string
```
I noticed that `spotify/docker-client` uses the same object:
https://github.com/spotify/docker-client/blob/da5c99d98ffdeaa9a537ffe343cb11bd900e1a84/src/main/java/com/spotify/docker/client/messages/Container.java#L90
https://github.com/spotify/docker-client/blob/da5c99d98ffdeaa9a537ffe343cb11bd900e1a84/src/main/java/com/spotify/docker/client/messages/ContainerInfo.java#L136